frontends/main/src/app/getQueryClient.ts

```diff
@@ -0,0 +1,18 @@
+// https://tanstack.com/query/v4/docs/framework/react/guides/ssr#using-hydrate
+
+import { QueryClient } from "api/ssr"
+// import { cache } from "react"
+
+/*
+ * Using cache()  Errors with:
+ * Error occurred prerendering page "/about". Read more: https://nextjs.org/docs/messages/prerender-error
+      Error: Not implemented.
+    at Object.getCacheForType
+    https://github.com/vercel/next.js/issues/57205
+ */
+// const getQueryClient = cache(() => new QueryClient())
```

Comment:
The cached version worked fine for on the about page (and elsewhere so far).

Suggest adding:
```ts
{
    defaultOptions: {
      queries: {
        refetchOnReconnect: false,
        refetchOnWindowFocus: false,
        staleTime: 1000 * 60 * 60 * 24, // 24 hours
      },
    },
  }
```
To the query client options.

Currently it refetches on ever render (0 "stale time"). 

Refetching on tab refocus doesn't seem necessary for our site.